syntax = "proto3";

package hello_service;


message HelloRequest {
	string r1 = 1;
	string r2 = 2;
}

message HelloReply {
	string message =1 ;
}

service Hello {
	rpc sayHello(HelloRequest)
		returns (HelloReply) {}
}

